docs/src/main/sphinx/sql/alter-materialized-view.rst

```diff
@@ -31,6 +48,19 @@ Rename materialized view ``people`` to ``users``, if materialized view
 
     ALTER MATERIALIZED VIEW IF EXISTS people RENAME TO users;
 
+Set view properties (``x = y``) in materialized view ``people``::
+
+    ALTER MATERIALIZED VIEW people SET PROPERTIES x = 'y';
```

Comment:
Do we have some mechanism (a system table like `system.metadata.table_properties`?) to inspect available properties and their data types and default values?

Reply:
https://github.com/trinodb/trino/issues/10916